container-guide/concepts/containers-podman.xml

```diff
@@ -403,14 +403,33 @@
 <screen>&prompt.user;podman run --rm -ti suse/sle15</screen>
 <note>
 <title>
-Solving network connection issues with rootful containers
+Fixing the lost network access to rootful containers
 </title>
 <para>
-Rootful containers may occasionally lose network connectivity due to another process deleting the firewall rules configured by &netavark;. Starting with version 1.9.0, &netavark; comes with the <systemitem>netavark-firewalld-reload.service</systemitem> service that solves the problem. Enable and start the service as follows:
+Rootful containers may occasionally become inaccessible from the outside. The
+issue is caused by firewalld reloading its permanent rules and discarding any
+temporary rules created by &podman;'s networking backend (either CNI or
+&netavark;).
+
+A temporary workaround is to reload the &podman; network using the
+<command>podman network reload --all</command> command. If you are using
+&netavark; as the network backend, then as of version 1.9.0 of &netavark, you
+can use the <systemitem>netavark-firewalld-reload.service</systemitem> service
+to solve this issue. Enable and start the service as follows:
 </para>
 <screen>&prompt.root;systemctl enable netavark-firewalld-reload.service
 &prompt.root;systemctl restart netavark-firewalld-reload.service
 </screen>
+You can check which backend and version you are using by running
+<replaceable>podman info --format "{{.Host.NetworkBackend}}"</replaceable> and
+<replaceable>podman info --format
+"{{.Host.NetworkBackendInfo.Version}}"</replaceable>, respectively.
+
+We recommend to add permanent firewall rules for containers you want to be
```

Comment:
Is this about users running containers with long-running jobs without frequently stopping and rerunning containers?

Reply:
Yes, for instance for a webpage. Such a container would be running all the time and it makes then sense to add a permanent firewall rule for it.